src/database/DataTypes/Requisition.js

```diff
@@ -469,15 +481,31 @@ export class Requisition extends Realm.Object {
     return this.isRequest ? this.canFinaliseRequest : this.canFinaliseResponse;
   }
 
+  /**
+   * Get if order type is emergency
+   * @returns {boolean}
+   */
+  get isEmergencyOrder() {
+    const tags = this.isRequest ? this.storeTags : this.customerTags;
+    return this.program?.getOrderType?.(this.orderType, tags)?.isEmergency;
+  }
+
   /**
    * Finalise this requisition.
    *
    * @param  {Realm}  database
    */
   finalise(database) {
-    this.pruneRedundantItems(database);
-    this.status = 'finalised';
+    const requisitionWithZeroQuantity = UIDatabase.getPreference(
```

Comment:
Nit-picking but this is a pref and should say `isKeepRequisitionWithZeroQuantityActive` or something.

Reply:
@sworup should be fixed by https://github.com/openmsupply/mobile/pull/5104/commits/f5662cae36b543ea6487e44ccbcfb51690aef840